<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zjm</class>
 <widget class="QWidget" name="zjm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zjm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>290</y>
     <width>189</width>
     <height>123</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>途经城市</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>北京</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>天津</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>郑州</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>西安</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>南京</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>上海</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>武汉</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>长沙</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>成都</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color: rgb(255,165,101);</string>
        </property>
        <property name="text">
         <string>广州</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>1060</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>采用</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>1060</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重新生成</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-radius:10px;
background: rgb(255,179,167);</string>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-radius:10px;
background: rgb(255,179,167);</string>
   </property>
   <property name="text">
    <string>策略</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Adobe Arabic&quot;;
background-color: rgb(255,185,90);
border-radius:10px;</string>
   </property>
   <item>
    <property name="text">
     <string>User1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-radius:10px;
background: rgb(255,179,167);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>480</y>
     <width>46</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46,169,223);</string>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
background: rgb(255,179,167);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>更换账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>480</y>
     <width>66</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>时间限定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>208</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,185,90);
font: 8pt &quot;Adobe Arabic&quot;;
border-radius:10px;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>郑州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西安</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>南京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>武汉</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>长沙</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成都</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广州</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起点</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>253</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,185,90);
font: 8pt &quot;Adobe Arabic&quot;;
border-radius:10px;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>郑州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西安</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>南京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>武汉</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>长沙</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成都</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广州</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>783</x>
     <y>433</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,185,90);
font: 8pt &quot;Adobe Arabic&quot;;
border-radius:10px;</string>
   </property>
   <item>
    <property name="text">
     <string>最少时间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最少费用</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>规定时间最少费用</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>480</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>小时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Adobe Arabic&quot;;
border-style:outset;
border-radius:10px;
background: rgb(255,179,167);</string>
   </property>
   <property name="text">
    <string>确定选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>旅行策略</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>630</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>630</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/D:/Downloads/QQdownload/oz.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_8</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>spinBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>label_6</zorder>
  <zorder>frame</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
